### PR TITLE
edits up to section II

### PR DESCRIPTION
**Pull request by wesleypchan**
Krish

for Section II. C. Future plans. 
you said "One very important aspect of autonomous systems which we like to implement is the ability of the UAV to recover from erroneous decision, false positives that might result in highly cluttered and unstructured scene."

can you add in a sentence or two to mention how you will do this? Just brief ideas is fine.

### CODE REVIEW

File: root.tex [hunk 1]
```diff
@@ -56,10 +55,10 @@
...
 
-\section{Project Personnel}
-Team JSK is made of 11 members. The 11 members are Prof. Inaba, Prof. Okada, Dr. Kakiuchi, Dr. W. Chan, Bakui Chou, Xiangyu Chen, Krishneel Chaudhary, Kohei Kimura, Yuki Furuta and Hiroto Mizohana. The team is divided into 3 groups generally to handle various tasks such as software and hardware developments and other project related activities. All team members are from JSK Lab, in The University of Tokyo.
+\subsection{Project Personnel}
+Team JSK is made of eleven members: Prof. Masayuki Inaba, Prof. Kei Okada, Dr. Yohei Kakiuchi, Dr. Wesley Chan, Bakui Chou, Xiangyu Chen, Krishneel Chaudhary, Kohei Kimura, Yuki Furuta, and Hiroto Mizohana. The team is roughly divided into 3 groups corresponding to each task to handle software and hardware develThen just remove the email address and give the JSK websiteopments and other project related activities.
```
Comment (iKrishneel) [inline root.tex:61]: @wesleypchan  Please check the last sentence of this line. Its very very ambiguous.

Comment (iKrishneel) [inline root.tex:61]: I can i corrected it. 

deleted this : Then just remove the email address and give the JSK website

**Comment by iKrishneel** [comment]
k i updated future works

File: root.tex (hunk 1 above)
Comment (wesleypchan) [inline root.tex:61]: That was a mistake. Thanks for correcting it

**Pull request merged**